src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -152,19 +137,25 @@ class HttpGoogleServicesDAO(
           //   workspace writer - object reader
           //   workspace reader - object reader
           //   bucket service account - object owner
-          val defaultObjectAcls = groupAccessLevelsAscending.map(ac => newObjectAccessControl(makeGroupEntityString(toGroupId(bucketName, ac)), "READER")) :+
-            newObjectAccessControl("user-" + serviceAccountClientId, "OWNER")
+          val defaultObjectAcls =
+            groupsByAccess.map { case (access, group) => newObjectAccessControl(makeGroupEntityString(group.groupEmail.value), "READER") }.toSeq :+
+              newObjectAccessControl("user-" + serviceAccountClientId, "OWNER")
 
           val bucket = new Bucket().setName(bucketName).setAcl(bucketAcls).setDefaultObjectAcl(defaultObjectAcls)
           val inserter = getStorage(getBucketServiceAccountCredential).buckets.insert(projectId, bucket)
           blocking {
             inserter.execute
           }
+          GoogleWorkspaceInfo(bucketName, groupsByAccess)
         }
       }
     }
 
-    val doItAll = insertGroups(workspaceName, bucketName) flatMap assertSuccessfulTries flatMap insertOwnerMember flatMap insertBucket
+    val groupRefsByAccess: Map[WorkspaceAccessLevel, RawlsGroupRef] = groupAccessLevelsAscending.map { accessLevel =>
+      (accessLevel, RawlsGroupRef(RawlsGroupName(s"${bucketName}-${accessLevel.toString}")))
```

Comment:
`toGroupId` is still used by `newGroup` / `setupWorkspace`.  Since this is a similar use of a "magic string" I would like this string creation to also be in a separate function.